src/View/mainpage/StructureTab.py

```diff
@@ -25,19 +25,17 @@ class StructureTab(QtWidgets.QWidget):
 
     def __init__(self, moving=False):
         QtWidgets.QWidget.__init__(self)
-        if moving:
-            self.patient_dict_container = MovingDictContainer()
-        else:
-            self.patient_dict_container = PatientDictContainer()
+        self.patient_dict_container = PatientDictContainer()
+        self.moving_dict_container = MovingDictContainer()
         self.rois = self.patient_dict_container.get("rois")
-        self.color_dict = self.init_color_roi()
+        self.color_dict = self.init_color_roi(self.patient_dict_container)
         self.patient_dict_container.set("roi_color_dict", self.color_dict)
         self.structure_tab_layout = QtWidgets.QVBoxLayout()
 
-        self.roi_delete_handler = ROIDelOption(self.structure_modified)
-        self.roi_draw_handler = ROIDrawOption(self.structure_modified)
+        self.roi_delete_handler = ROIDelOption(self.fixed_container_structure_modified)
```

Comment:
This and the next line are too long for PEP 8. So is line 80, 223, 325, 538, and 612.